apps/web/pages/usernames.tsx

```diff
@@ -0,0 +1,52 @@
+import Modal from 'apps/web/src/components/Modal';
+import Head from 'next/head';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useCallback, useState } from 'react';
+import { InformationCircleIcon } from '@heroicons/react/20/solid';
+
+enum ClaimProgression {
+  SEARCH,
+  CLAIM
+}
+
+export default function Usernames() {
+  const [progress, setProgress] = useState<ClaimProgression>(ClaimProgression.SEARCH)
+  const [learnMoreModalOpen, setLearnMoreModalOpen] = useState(false)
+  const toggleModal = useCallback(() => setLearnMoreModalOpen(open => !open), [])
+  const selectName = useCallback(() => {
+    setProgress(ClaimProgression.CLAIM)
+  }, [])
+  return (
+    <>
+      <Head>
+        <title>Base | Usernames</title>
+        <meta
+          content="Base is a secure, low-cost, builder-friendly Ethereum L2 built to bring the next billion users onchain."
+          name="description"
+        />
+      </Head>
+      <main className="flex w-full flex-col items-center bg-white">
+        <input type="text" placeholder='search for a username'/>
+        <button type='button' onClick={selectName}>option: name1</button>
+
+        {progress === ClaimProgression.CLAIM && <p>
+          unlock your username for free! <button type='button' className="underline" onClick={toggleModal}>learn more</button>
```

Comment:
This is placeholder content that will be addressed when we build a real search UI.